great-idea/css/index.css

```diff
@@ -63,4 +63,196 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+nav{
+	display:flex;
+	justify-content:space-around;
+	margin-top:40px;
+}
+a{
+	color:grey;
+	background-color:white;
+	text-decoration:none;
+}
+h1{
+	font-size:90px;
+	margin-top:130px;
+	text-align:center;
+}
+.code{
+	margin:100px;
+	
+}
+.big{
+	display:flex;
+	justify-content:center;
+	border-bottom:1px solid black;
+}
+.top{
+	display:flex;
+	justify-content:center;
+	margin:40px;
+	max-width:75%;
+	margin-left:200px;
+	
+	
+	
+}
+.top .features{
+	display:flex;
+	padding:80px;
+	margin:100px;
+	
+	
+}
+.top .about{
+	display:flex;
+	padding:80px;
+	margin:100px;
+}
+.middle-img{
+	width:100%;
+	padding:80px;
+}
+.bottom{
```

Comment:
For this you'd want to put the margin in one line since you are including margin for more than one side. It would be `margin: 40px 40px 40px 200px`